src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+    class="big-form form"
+    data-qa="big">
+      <label for="big-search">
+        <img src="images/Search.svg"
+        alt="Search"
+        class="big-image">
+        <input type="search"
+        class="big-search search"
+        name="big-search"
+        id="big-search"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
+    <form action="#"
+    class="small-form form"
+    data-qa="small">
+      <img src="images/Search.svg"
+      alt="Search"
+      class="small-image">
+      <input type="search"
+      class="small-search search"
+      name="small-search"
+      id="small-search"
+      placeholder="Try &quot;Los Angeles&quot;">
+    </form>
+    </div>
   </body>
 </html>
```

Comment:
```suggestion
</html>

```
add an empty line to the end of file